skypy/power_spectrum/_eisenstein_hu.py

```diff
@@ -84,7 +84,7 @@ def transfer_with_wiggles(wavenumber, A_s, n_s, cosmology, kwmap=0.02):
         raise ValueError("Tcmb0 for input cosmology must be non-zero if"
                          "wiggles = True")
 
-    ak = wavenumber * h0
+    ak = wavenumber
```

Comment:
Can I use the camb module results to do the comparison in the tests?
Or should it always be against an external source?

Reply:
That's a good question. Ideally we will have some validation plots where we'll compare with external data.

Reply:
Should I leave the values I used for the tests?
They do come from cosmosis but it uses essentialy the same code.

Reply:
I'd say leave them if they work (unless @rrjbca has a different opinion). We can worry about modifying the tests later if necessary

Reply:
The change to use astropy constants caused those values to fail the `np.allclose` test.
They are still close.

```python
array_w = eisenstein_hu(wavenumber, A_s, n_s, Planck15, kwmap=0.02, wiggle=True)
array_w = [2.99126326e+04, 1.32023496e+05, 1.80797616e+04, 1.49108261e+02, 4.53912529e-01]
pk_cosmosis_w  = [2.99156428e+04, 1.32036782e+05, 1.80815810e+04, 1.49123267e+02, 4.53958208e-01]
```
What is the policy to configure `np.allclose`?

Don't really see the point of comparing these kind of outputs given that they are **the same** code, just placed somewhere else.
Comparing against camb with a well thought tolerance makes more sense to me

Reply:
hmm I see. How close are they? Is it a factor `c` difference?

The thing is this PR is to fix the `h` issue and the amplitude scaling. That's what I meant about worrying about the tests later. But if it's a factor `c` that's causing the code to fail, you could possibly update the values. What do you think?

Reply:
It is just a tiny numerical difference caused by the precission of the speed of light constant.
The above example shows they are within 0.5% from each other.

Reply:
I see. You can then play around with the `rtol` and `atol` parameters in `np.allclose`, so it passes the tests and then open a followup issue?

Reply:
Since we have changed the code due to a bug in the original code (low precision of c) I don't think it's appropriate to keep the old (wrong) cosmosis values in the test. A simple suggestion would just be to calculate the expected values using the patched skypy implementation (not cosmosis). A bit unsatisfying comparing the code with itself but we have independently plotted the outputs and validated them by eye against CAMB/CLASS. I can't think of a better option within the scope of this PR.